### PR TITLE
8163921: HttpURLConnection default Accept header is malformed according to HTTP/1.1 RFC

### PR DESCRIPTION
**Pull request by djelinski**
Fix RFC compliance.
Tier1 and tier2 passed.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8163921](https://bugs.openjdk.java.net/browse/JDK-8163921): HttpURLConnection default Accept header is malformed according to HTTP/1.1 RFC

### Reviewers
 * [Daniel Fuchs](https://openjdk.java.net/census#dfuchs) (@dfuch - **Reviewer**) ⚠️ Review applies to 86d6d6d1a019164547dcd0226e564bf0af7d3b1a
 * [Michael McMahon](https://openjdk.java.net/census#michaelm) (@Michael-Mc-Mahon - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7181/head:pull/7181` \
`$ git checkout pull/7181`

Update a local copy of the PR: \
`$ git checkout pull/7181` \
`$ git pull https://git.openjdk.java.net/jdk pull/7181/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7181`

View PR using the GUI difftool: \
`$ git pr show -t 7181`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7181.diff">https://git.openjdk.java.net/jdk/pull/7181.diff</a>

</details>

### CODE REVIEW

**Comment by dfuch** [comment]
Could you update the copyright years and add `8163921` to the `@bug` tag in the test?

**Review by dfuch — APPROVED** [review]
LGTM. @Michael-Mc-Mahon what's your take?

File: src/java.base/share/classes/sun/net/www/protocol/http/HttpURLConnection.java [hunk 1]
```diff
@@ -289,7 +289,7 @@ private static String getNetProperty(String name) {
 
     static final String httpVersion = "HTTP/1.1";
     static final String acceptString =
-        "text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2";
+        "text/html, image/gif, image/jpeg, */*; q=0.2";
```
Comment (Michael-Mc-Mahon) [inline src/java.base/share/classes/sun/net/www/protocol/http/HttpURLConnection.java:293]: It looks syntactically correct now, but I wonder about the actual semantics, and whether it makes any sense to be asking for 80% reduction in quality of other media types.

Comment (Michael-Mc-Mahon) [reply src/java.base/share/classes/sun/net/www/protocol/http/HttpURLConnection.java:293]: Should we just have `static final String acceptString = "*/*";`

**Review by Michael-Mc-Mahon — APPROVED** [review]
LGTM

**Pull request closed without merging**